src/graphnet/models/task/classification.py

```diff
@@ -14,7 +38,7 @@ class BinaryClassificationTask(Task):
 
     def _forward(self, x: Tensor) -> Tensor:
         # transform probability of being muon
-        return torch.sigmoid(x)
+        return sigmoid(x)
```

Comment:
Change is not pertinent to the goal of the PR.
```suggestion
        return torch.sigmoid(x)
```